examples/network/exampleApplications/neighbourhoodHighlight.html

```diff
@@ -99,16 +98,20 @@
         }
       }
       var connectedNodes = network.getConnectedNodes(selectedNode);
-      var allConnectedNodes = [];
+      var allConnectedNodes = connectedNodes.slice(0);
+      var allConnectedNodesTemp = connectedNodes.slice(0);
 
-      // get the second degree nodes
+      // get the n degree nodes
       for (i = 1; i < degrees; i++) {
-        for (j = 0; j < connectedNodes.length; j++) {
-          allConnectedNodes = allConnectedNodes.concat(network.getConnectedNodes(connectedNodes[j]));
+        for (j = 0; j < allConnectedNodes.length; j++) {
```

Comment:
I think it's possible to avoid the use of a temporary array, simply by remembering the initial length:

```
var length = allConnectedNodes.length;
for (j = 0; j < length; j++) {
...
```
I would prefer if the code did this, and `allConnectedNodesTemp` was removed.